pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py

```diff
@@ -689,17 +689,21 @@ def sort_legal_provision(elem):
         return sort_title, sort_number
 
     @staticmethod
-    def sort_hints_laws(elem):
+    def sort_by_index(elem):
         """
-        Provides the sort key for the supplied hint & law element as a tuple consisting of:
+        Provides the sort key for the supplied hint / law / legal provision
+        element as a tuple consisting of:
          * index
 
         Args:
-            elem (dict): one element of the hints list
+            elem (dict): one element of the hints / laws / legal provisions list
         Returns:
             sort key (tuple)
         """
-        return elem.get('index', 1000)
+        index = elem.get('Index', 1000)
+        if index is None:
```

Comment:
I think this "if" is redundant to above (there is a default value for the get, so index can't be None after that)

Reply:
Geolink formatter might return a None value: https://github.com/openoereb/geolink_formatter/blob/fdad0e3573b4492c2182c6d539ba5df88185c79b/geolink_formatter/entity.py#L91 (Index defined but none). But you are right, that is not according to the specs.

Reply:
Ah, good point.
Can you please add that as well as a comment to the code? (I didn't think about the case where index was explicitly set to "None" somewhere in the program).